### ISSUE TITLE
1.3.1: It's not just about 'structure and relationships'. It's also about 'information'

### ISSUE THREAD

**Issue by jfhector** [theappbusiness/accessibility-guidelines#104]
Information conveyed through presentation should be available to screen readers or available in text, as per [WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).

#### Issue

The [headline and the summary of SC 1.3.1](https://theappbusiness.github.io/accessibility-guidelines/guidelines/1.3.1.html) don't mention anything about that fact.
